.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,14 @@
 blank_issues_enabled: false
 contact_links:
-  - name: Helper-Scripts Community Support
-    url: https://github.com/community-scripts/ProxmoxVE/discussions
-    about: Please ask and answer questions here.
+    - name: 📃 Documentation Issue
+      url: https://github.com/community-scripts/ProxmoxVE/issues/new?assignees=&labels=Status%3A+Unconfirmed&projects=&template=frontend_report.yml&title=%5BFrontend%5D%3A+
+      about: This issue tracker is not for documentation issues. Please file documentation issues here.
```

Comment:
This sentence is confusing and contradicting. Do we need a separate Documentation Issue? Wouldn't the previous `website issue` cover both frontend and documentation. Not sure users will bother to separate these issues when it's all on the website.